### PR TITLE
Fixes resources with variable name

### PR DESCRIPTION
**Pull request by petems**
Lifted heavily from https://github.com/russCloak/SublimePuppet/pull/20

Before:
screenshot 2016-03-11 15 38 11

After:
screenshot 2016-03-11 15 42 11

### CODE REVIEW

**Comment by 50Wliu** [comment]
@petems Are you sure you referenced the correct issue?  In addition, does #25 cover this already?

**Comment by petems** [comment]
@50Wliu Yep, it's still an issue, here's a screenshot of the current look (4 different highlights):
screenshot 2016-03-11 15 31 45

I'm fixing the space one in another PR

File: grammars/puppet.cson [hunk 1]
```diff
@@ -123,6 +123,15 @@
     'name': 'meta.definition.resource.puppet'
   }
   {
+    'captures':
+      '1':
+        'name': 'storage.type.puppet'
+      '2':
+        'name': 'entity.name.section.puppet'
+    'match': '^\\s*(\\w+)\\s*{\\s*(\\$[a-zA-Z_]+)\\s*:'
```
Comment (50Wliu) [inline grammars/puppet.cson]: Please move the `match` above `captures`

Comment (petems) [inline grammars/puppet.cson]: Done :+1:

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**